tests/util/test_file.cpp

```diff
@@ -101,9 +100,25 @@ SyncTestFile::SyncTestFile(SyncServer& server,
     if (realm_schema)
         schema = std::move(realm_schema);
 
-    sync_config = std::make_shared<SyncConfig>(SyncManager::shared().get_user({ "user", url }, "not_a_real_token"), url);
+    sync_config = std::make_shared<SyncConfig>(SyncManager::shared().get_user({ "test", url }, "not_a_real_token"), url);
+    sync_config->user->set_is_admin(true);
     sync_config->stop_policy = SyncSessionStopPolicy::Immediately;
-    sync_config->bind_session_handler = [=](auto&, auto& config, auto session) { session->refresh_access_token(s_test_token, config.realm_url()); };
+    sync_config->bind_session_handler = [=](auto&, auto& config, auto session) {
+        std::string token, encoded;
+        // Tokens without a path are considered admin tokens by the sync service, so for
+        // non-admin users we need to add a path
```

Comment:
Add a FIXME about the fact that this is likely to change within the next few weeks.